src/app.ts

```diff
@@ -849,6 +851,18 @@ router.get(
   safeRequestHandler(accountOpeningRequestAPI.retrieveAccountOpeningRequest)
 );
 
+// COMMERCIAL REGISTRATIONS
+
+router.get(
+  "/commercial_registrations/search_by_name",
+  safeRequestHandler(search)
+);
+
+/**
+ * This endpoint has an associated cost per request. Contact your Partner Manager for more information.
```

Comment:
could you remove these solaris notes? they are not relevant

Reply:
Yes: https://github.com/kontist/mock-solaris/pull/197/commits/180162866532123284dea8476c9ddb09d1410eda